packages/input-info/src/ux-input-info.ts

```diff
@@ -41,4 +44,20 @@ export class UxInputInfo implements UxComponent {
       this.target = inputElement.au.controller.viewModel;
     }
   }
+
+  @computedFrom('target', 'target.maxlength')
+  public get maxLength(): number {
+    if (this.target instanceof UxInput || this.target instanceof UxTextArea) {
+      return this.target.maxlength;
+    }
+    return 0;
+  }
+
+  @computedFrom('target', 'target.value', 'target.value.length')
```

Comment:
maybe just only do against `target.value.length`?

Reply:
actually if `target.value` is a string, we only need to have `computed` against `target.value`

Reply:
I will test this :-)

Reply:
@bigopon 
Fix: https://github.com/platform5/ux/commit/854454dc1645cdce4ade85fe062acb8350f77cf4

Reply:
I think you only need to have:
```ts
  @computedFrom('target.value')
  public get length(): number {
    ...
  }
```

Reply:
@bigopon https://github.com/aurelia/ux/pull/232/commits/2c4135b2664ce261fd39db7143add1f29b6cd88e